insights/parsers/systemd/config.py

```diff
@@ -125,7 +128,39 @@ def __setitem__(self, key, value):
 
 
 def parse_systemd_ini(content):
-    """Function to parse config format file, the result format is dictionary"""
+    """Function to parse config format file, the result format is dictionary
+     When there are multi values for one key, the values are formatted as list. When there are multi lines in one value, these lines are formatted as one line.
+     Example::
+
+        [Service]
+        Type=notify
+        EnvironmentFile=-/etc/sysconfig/docker
+        EnvironmentFile=-/etc/sysconfig/docker-storage
+        EnvironmentFile=-/etc/sysconfig/docker-network
+        ExecStart=/bin/sh -c '/usr/bin/docker-current daemon \\
+          --authorization-plugin=rhel-push-plugin \\
+          --exec-opt native.cgroupdriver=systemd \\
+          $OPTIONS \\
+          $DOCKER_STORAGE_OPTIONS \\
+          $DOCKER_NETWORK_OPTIONS \\
+          $ADD_REGISTRY \\
+          $BLOCK_REGISTRY \\
+          $INSECURE_REGISTRY \\
+          2>&1 | /usr/bin/forward-journald -tag docker'
+        ...
+
+    Parse Result::
+
+        {
+            "Service":{
+                "Type": "notify",
+                "EnvironmentFile": ["-/etc/sysconfig/docker", "-/etc/sysconfig/docker-storage", " -/etc/sysconfig/docker-network"],
+                "ExecStart": "/bin/sh -c '/usr/bin/docker-current daemon --authorization-plugin=rhel-push-plugin --exec-opt native.cgroupdriver=systemd $OPTIONS $DOCKER_STORAGE_OPTIONS ... 2>&1 | /usr/bin/forward-journald -tag docker'"
+                ...
+            }
+        }
+    """
```

Comment:
Please refer to the documentation guidelines on how to document sample input and examples: http://insights-core.readthedocs.io/en/1.x/docs_guidelines.html#documentation-guidelines